### PR TITLE
athei: retain at rank III

### PR DESCRIPTION
**Pull request by athei**
[Rendered Version](https://github.com/athei/Evaluations/blob/at/eval/evidence/athei/0001-retain-III.md)

### CODE REVIEW

File: evidence/athei/0001-retain-III.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+| **Report Date** | Date of submission (2024/08/26)                                                             |
+| **Submitted by**| Alexander Theißen                                                                           |
+
+
+## Member details
+
+- Matrix username: @alex:parity.io
+- Polkadot address: 15db5ksZgmhWE9U8MDq4wLKUdFivLVBybztWV8nmaJvv3NU1
+- Current rank: III
+- Date of initial induction: 2024/03/07
```
Comment (anaelleltd) [inline evidence/athei/0001-retain-III.md:14]: Date of initial induction should read: "Seeding", in reference to: https://github.com/polkadot-fellows/seeding/pull/17.

**Comment by anaelleltd** [comment]
Please take care not to remove any part of the template, even if not used for this submission.
The part 👇below was removed from your report.
image

File: evidence/athei/0001-retain-III.md (hunk 1 above)
Comment (athei) [reply evidence/athei/0001-retain-III.md:14]: Are you sure? I thought this refers to the induction into the core fellowship. Not into the collective.

**Comment by athei** [comment]
Added the section.

File: evidence/athei/0001-retain-III.md (hunk 1 above)
Comment (anaelleltd) [reply evidence/athei/0001-retain-III.md:14]: Yes, this is about induction to the collective, though it is mostly relevant for promotion requests.

**Pull request merged**